packages/webapp/public/locales/en/translation.json

```diff
@@ -908,15 +856,12 @@
       "INSECTS": "Insects",
       "LOADING": {
         "BODY": "We're generating the latest biodiversity insights for your farm. This can take up to 60 seconds.",
-        "PREVIEW": "Loading...",
         "TITLE": "Generating the latest biodiversity insights..."
       },
       "MAMMALS": "Mammals",
       "PLANTS": "Plants",
-      "SPECIES_COUNT": "{{count}} species",
       "SPECIES_COUNT_one": "{{count}} species",
```

Comment:
This confused the heck out of me but that is a cool feature of i18n I didn't know about!